.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -18,7 +18,20 @@ labels: 'bug, needs triage'
 
 ## 🐛 Bug Report
 
-(A clear and concise description of what the bug is)
+### Prerequisites
+
+<!--
+IMPORTANT: Please check the following items before creating a issue.
+Mark the checkbox by putting an X between the bracket.
+This way we know you've done these steps first.
+-->
+
+- [ ] I'm using the latest version of Docusaurus.
+- [ ] I have tried the `yarn clear` command.
```

Comment:
```suggestion
- [ ] I have tried the `npm run clear` or `yarn clear` command.
- [ ] I have tried `rm -rf node_modules yarn.lock package-lock.json` and re-installing packages
```

I think showing the npm command might be useful for non-frontend-dev users that might be confused by seeing a yarn command.

Also think re-generating lockfile / reinstall is a useful thing to try